topics/qt/index.md

```diff
@@ -0,0 +1,15 @@
+---
+aliases: qt-framework, cute, qt-application
```

Comment:
Looking at https://github.com/topics/cute, the repositories using that topic don't seem related to Qt. Can you remove `cute` from this list?

I see https://github.com/topics/qt5 is pretty popular. What about adding `qt5` to this list?